navigator_engine/common/graph_loader.py

```diff
@@ -201,3 +215,25 @@ def _map_excel_boolean(boolean):
     else:
         raise ValueError('Value {} read from Initial Graph Config is not valid; Only TRUE and FALSE are valid values.'
                          .format(boolean))
+
+
+def _parse_resources(resource_cell):
+    resources = []
+
+    if pd.isnull(resource_cell):
+        return resources
+
+    resource_rows = re.split('\n', resource_cell)
+    for row in resource_rows:
+        url_re = re.compile(r'(https?://[^\s]+)')
+        search_results = url_re.search(row)
+        url = search_results.group(0)
+        title = row[0:search_results.span()[0]].strip()
+
+        try:
+            url_parsed = urlparse(url)
+            resources.append({'title': title, 'url': url_parsed.geturl()})
+        except ValueError as e:
+            logging.warning(f'{url} is not a valid URL, omitting resource')

```

Comment:
please use:
```
logger = logging.getLogger(__name__)
logger.warning("...")
```
Also in this case I think it should be logger at least as an error. If the URL parsing fails it means we have a bad graph structure which should be fixes asap.

Reply:
We won't get bad graph structure but bad URL's defined in the input spreadsheet won't load. That would still be bad and I've made the changes requested.

Reply:
I agree that we should get a logger before logging. 

I think a warning is fine in this situation as it doesn't break the graph structure, it's only a small content issue that can be resolved later.